internal/bundle/bind_resource_test.go

```diff
@@ -101,12 +101,12 @@ func TestAccAbortBind(t *testing.T) {
 		destroyBundle(t, ctx, bundleRoot)
 	})
 
-	t.Setenv("BUNDLE_ROOT", bundleRoot)
-	c := internal.NewCobraTestRunner(t, "bundle", "deployment", "bind", "foo", fmt.Sprint(jobId))
-
-	// Simulate user aborting the bind. This is done by not providing any input to the prompt in non-interactive mode.
-	_, _, err = c.Run()
-	require.ErrorContains(t, err, "failed to bind the resource")
+	// Bind should fail because prompting is not possible.
+	stdout, stderr, err := blackBoxRun(t, bundleRoot, "bundle", "deployment", "bind", "foo", fmt.Sprint(jobId))
+	require.Error(t, err)
+	require.Contains(t, stderr, "failed to bind the resource")
+	require.Contains(t, stderr, "This bind operation requires user confirmation, but the current console does not support prompting. Please specify --auto-approve if you would like to skip prompts and proceed.")
+	require.Equal(t, "", stdout)
```

Comment:
Why do you need to use the black-box runner if the underlying was fixed?

Reply:
There's a bug in `cmdio.IsPromptSupported` where it returns true when you use the `debug test` functionality in VSCode. I would like to not spend my time right now to figure out why that's the case since this could be a rabbit hole.

Normally I use `t.Setenv("TERM", "dumb")` to make `cmdio.IsPromptSupported` return false but I'd rather not rely on it if possible.

Reply:
Fair point about debugging though. I can flip back to using `t.Setenv("TERM", "dumb")`.

Reply:
Another signal though that we should spend some time on our prompting library and build some understanding.